bin/nbp/nbplkup.c

```diff
@@ -76,7 +78,7 @@ int main(int ac, char **av)
     set_charset_name(CH_MAC, MACCHARSET);
 
     memset(&addr, 0, sizeof(addr));
-    while ((c = getopt(ac, av, "r:A:m:")) != EOF) {
+    while ((c = getopt(ac, av, "r:A:m:s")) != EOF) {
```

Comment:
The convention for oparg strings in this codebase is: alphabetic order, with bool options first and parameterized options second. Makes for more readable, less error-prone code. (I dealt with some obnoxious bugs in the past because of this.)

In this case, the ideal optarg string would be: `"sA:m:r:"`

Since the number of options is still small, there's no rush to organize this one. But perhaps next time an option gets added. :)

Reply:
Might as well do it while I'm here!